packages/components/skeleton/register.ts

```diff
@@ -0,0 +1,9 @@
+import { Skeleton } from './src/skeleton';
```

Comment:
Add `.js` extension plz. This helps when you run the code directly in the browser (without a bundler).